src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.task.TaskList;
+import duke.util.DukeException;
+import duke.util.Storage;
+
+/**
+ * Abstract class comprising information required to change the state of duke when executed.
+ */
+public abstract class Command {
+    /**
+     * Abstract method that performs a series of action to change the state of Duke
+     * and returns a message depending on the command type.
```

Comment:
Perhaps could shorten the method description, since it is supposed to be a short summary.